extension/wallet/src/lib.rs

```diff
@@ -225,10 +225,22 @@ pub async fn make_sell_create_swap_payload(
 pub async fn make_loan_request(
     wallet_name: String,
     collateral: String,
+    fee_rate: String,
+    timeout: String,
```

Comment:
What do you think of changing the API in a way that we always deal with days until we get back the loan response. Thatway Bobtimus has to the calculation to actual blockheight and we only need to confirm it again in the UI when we sign it?

Reply:
Yes! I should have done that in the first place. The absolute timelock being decided at the very start is not optimal, because the relative time may already decline while the user is still deciding on the loan. This calculating the timelock upon taking that is not an issue anymore :)

Reply:
will do this in a follow-up that includes baru upgrades (because to do this properly some minor changes in baru will be needed that I'll just do on the way)